.github/workflows/pull_request.yml

```
@@ -40,6 +40,8 @@ jobs:
           - os: depot-windows-2022
           - os: depot-ubuntu-24.04-arm-16
     steps:
+      - name: Free Disk Space
```

actionlint (1.7.12)

[error] step must run script with "run" section or run action with "uses" section

(syntax-check)